desktop/backend/python/controller/actions.py

```
@@ -0,0 +1,203 @@
+            (coords[i], coords[i + 1])
+            for i in range(0, len(coords), 2)
+        ]
+
+        path = self.mouse.draw_polyline(points)
+        self.mouse.queue_path(path)
+
+    # ========================
+    # Shared
+    # ========================
+
+    def _wait(self, tokens: List[str]):
```

Ruff (0.14.10)

Missing return type annotation for private function `_wait`

Add return type annotation: `None`

(ANN202)